Detectors/TPC/calibration/include/TPCCalibration/IDCGroupingParameter.h

```diff
@@ -38,7 +38,7 @@ struct ParameterIDCGroup : public o2::conf::ConfigurableParamHelper<ParameterIDC
   unsigned char GroupLastRowsThreshold[Mapper::NREGIONS]{3, 3, 3, 3, 2, 2, 2, 2, 2, 2}; ///< if the last group (region edges) consists in row direction less then mGroupLastRowsThreshold pads then it will be grouped into the previous group
   unsigned char GroupLastPadsThreshold[Mapper::NREGIONS]{3, 3, 3, 3, 2, 2, 2, 2, 1, 1}; ///< if the last group (sector edges) consists in pad direction less then mGroupLastPadsThreshold pads then it will be grouped into the previous group
   AveragingMethod Method = AveragingMethod::SLOW;                                       ///< method which is used for averaging
-
+  float Sigma = 3.f;                                                                    ///< sigma cut which can be used during the grouping for outlier filtering
```

Comment:
In structs, all public data member should start with lower case, as if they were function names:
https://rawgit.com/AliceO2Group/CodingGuidelines/master/naming_formatting.html?showone=Variable_Names#Variable_Names

Also for values above...